### PR TITLE
Remove the last bits of ABS from the source

### PR DESCRIPTION
**Pull request by taywrobel**
Reid missed a few instances of ABS that caused the
app to crash when running (despite compiling).  This
fix removes references to ABS from the XML files, and
from the About fragment.

### CODE REVIEW

File: SoundStream/res/menu/search_menu.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <item 
         android:id="@+id/search" 
         android:showAsAction="always|collapseActionView"
-        android:actionViewClass="com.actionbarsherlock.widget.SearchView"
```
Comment (reidbaker) [inline SoundStream/res/menu/search_menu.xml:6]: This probably needs to be com.appcompat.widget.SearchView

File: SoundStream/res/values/strings.xml [hunk 2]
```diff
@@ -80,14 +80,12 @@
     <string name="meet_label">Meet the Team</string>
     <string name="team">Reid Baker\nDavid Greenhalgh\nElizabeth Johnson\nJesse Rosalia\nTaylor Wrobel</string>
     <string name="thanks_label">Special Thanks</string>
-    <string name="thanks_ABS"><u>ActionBarSherlock</u></string>
```
Comment (reidbaker) [inline SoundStream/res/values/strings.xml:83]: We should replace these with Appcompat

File: SoundStream/src/com/thelastcrusade/soundstream/components/AboutFragment.java [hunk 3]
```diff
@@ -48,12 +48,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         final TextView 
         	repoLinkText = (TextView)v.findViewById(R.id.repo_link),
         	SlidingMenuLinkText = (TextView)v.findViewById(R.id.thanks_SlidingMenu),
-        	ABSLinkText = (TextView)v.findViewById(R.id.thanks_ABS),
```
Comment (reidbaker) [inline SoundStream/src/com/thelastcrusade/soundstream/components/AboutFragment.java:51]: appcompat

File: SoundStream/src/com/thelastcrusade/soundstream/components/AboutFragment.java [hunk 4]
```diff
@@ -48,12 +48,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
         	emailLinkText = (TextView)v.findViewById(R.id.email_link);
         
         listenToHttpLink(repoLinkText, getString(R.string.repo_link));
         listenToHttpLink(SlidingMenuLinkText, getString(R.string.sliding_menu_link));
-        listenToHttpLink(ABSLinkText, getString(R.string.ABS_link));
```
Comment (reidbaker) [inline SoundStream/src/com/thelastcrusade/soundstream/components/AboutFragment.java:56]: appcompat here as well

**Comment by reidbaker** [comment]
This should be abandoned

**Comment by taywrobel** [comment]
Abandoning, in favor of Reid's fix.

**Pull request closed without merging**